modules/nw-multus-ipam-object.adoc

```diff
@@ -310,4 +310,62 @@ pod/whereabouts-reconciler-p4sxw 1/1 Running 0 6s
 pod/whereabouts-reconciler-rvfdv 1/1 Running 0 6s
 pod/whereabouts-reconciler-svzw9 1/1 Running 0 6s
 daemonset.apps/whereabouts-reconciler 6 6 6 6 6 kubernetes.io/os=linux 6s
+----
+
+[id="nw-multus-creating-whereabouts-dualstack-config_{context}"]
+== Creating a configuration for assignment of dualstack IP addresses dynamically
+
+DualStack IP address assignment can be configured with the `ipRanges` parameter for:
+- IPv4 addresses
+- IPv6 addresses
+- multiple IP address assignment 
+
+.Procedure
+
+. Set type to whereabouts
+. Use `ipRanges` to allocate IP addresses.
+
+For example, defining the range as `192.168.10.1/24` excludes the first network address and the last broadcast address.
+
+
+[source,yaml]
+----
+cniVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  additionalNetworks:
+  - name: whereabouts-shim
+    namespace: default
+    rawCNIConfig: |-
+      {
+       "name": "whereabouts-dual-stack",
+       "cniVersion": "0.3.1,
+       "type": "bridge",
+       "ipam": {
+         "type": "whereabouts",
+         "ipRanges": [{
+           "range": "192.168.10.1/24"
+         }, {
+           "range": "2001:db8::/64"
+         }]
+       }
+      }
+
+
+----
+
+. Verify that all IP addresses are assigned
+
+Run the following two commands to ensure the IP adresses are assigned as meta-data.
```

Comment:
With these kind of configurations, there's kind of two steps -- defining a network configuration, and then associating it with a pod. So the user won't see anything until they also associate it with a pod.

So maybe we can link to: https://docs.openshift.com/container-platform/4.13/networking/multiple_networks/attaching-pod.html#nw-multus-add-pod_attaching-pod which shows how to attach a network. And then change it to like:

> After [attaching this additional network to a pod], verify the IP address assignment by running these two commands to ensure the IP addresses are assigned, and is present in the pod metadata in the  `k8s.v1.cni.cncf.io/network-status` annotation:

(where the square brackets indicates a link pointed to the above location in docs -- and of course, modified to fit your style guide)